### ISSUE TITLE
Player scaling

### ISSUE THREAD

**Issue by JanWerder** [davidmz/apng-js#15]
The player currently doesn't scale the content of the canvas to the width & height of the canvas. This is relevant if i.e. the content of the buffers is larger than the canvas it's supposed to be rendered on.
Can this be added, so that the canvas scales its content?

**Comment by davidmz** [comment]
I dont think that library should do it. The correct way to scale canvas is to always create canvas with the original APNG dimensions and scale it via CSS. 

The canvas width and height defines the on-page dimensions AND the internal canvas coordinates. Unlike images, where width and height affects only on-page sizes but the internal image size fully defined by the image data themself.

Scaling image inside of canvas may cause unwanted artifacts. You can see it on [demo page](https://davidmz.github.io/apng-js/): upload some APNG and type in console `document.querySelector("canvas").getContext('2d').scale(1.2, 1.2)`. You will see resized animation with visible artifacts.

**Comment by JanWerder** [comment]
From my testing, scaling it via CSS cuts the image rather than scaling it, because the drawImage on the canvas still operates with the original width and height of the frames.

**Comment by davidmz** [comment]
Could you show some minimal code?
